azavea/iow-boundary-tool
# Add Topographic basemap for Land & water layer

## Overview

Since we can't add individual vector tiles, we switch the basemap wholesale. ~~When the Land & water layer is enabled, and the default basemap is set to active, we show the Topographic layer. Otherwise we show the Light Grey Canvas layer. The Satellite tile shows that layer regardless of Land & water state.~~

Adds a new Land & water basemap tile. Removes the Street and Land & water data layers. Updates styling for polygon and Municipal Boundary layers.

Further styling updates to be done in #49 

Closes #47 

### Demo

~~https://user-images.githubusercontent.com/1430060/187044736-63a2ea79-6927-428d-8d48-d1ee0e6d9c25.mp4~~

https://user-images.githubusercontent.com/1430060/187742964-3205b9bf-20d2-4086-99a1-7965789ec1c9.mp4

## Testing Instructions

- Checkout this branch and `server`
- Go to http://localhost:4545/draw
- [x] Ensure you no longer see Streets and Land & water data layers
- [x] Ensure you see a Land & water basemap, and switching basemaps works as expected
- [x] Ensure the styling of the sidebar matches the layout here: https://www.figma.com/file/tJPZ0lkT5C3SAyS2g2R1aA/IoW---Boundary-Sync-Tool?node-id=498%3A14698

## Checklist
